src/main/java/gov/cms/mat/cql_elm_translation/service/EffectiveDataRequirementService.java

```diff
@@ -61,43 +36,19 @@ private RequestData createDefaultRequestData(String identifier, String cql) {
   }
 
   /**
-   * @param bundleResource Bundle resource
-   * @return BundleEntry which is of type Measure
-   */
-  public Optional<Bundle.BundleEntryComponent> getMeasureEntry(Bundle bundleResource) {
-    return bundleResource.getEntry().stream()
-        .filter(
-            entry ->
-                StringUtils.equalsIgnoreCase(
-                    "Measure", entry.getResource().getResourceType().toString()))
-        .findFirst();
-  }
-
-  /**
-   * @param r5Measure retrieved from measure bundle
-   * @param library measure library
-   * @param accessToken used by MadieLibrarySourceProvider to make calls to madie-fhir-services
+   * @param libraryDetails - instance of CqlLibraryDetails
+   * @param recursive- indicates if the data requirements gathered should be recursive
```

Comment:
Might be helpful to future us to leave a note of what "recursive" means in this context.